django/contrib/auth/backends.py

```diff
@@ -8,8 +8,11 @@ class ModelBackend(object):
     """
     Authenticates against settings.AUTH_USER_MODEL.
     """
+    def __init__(self, request=None):
```

Comment:
The ticket says to make request available in authentication backends. I thought it also applies to the ModelBackend and RemoteUserBackend. That is why `self.request = request` is assigned in `authenticate()`.